### ISSUE TITLE
Combine 2 methods?

### ISSUE THREAD

**Issue by IanaKhitsko** [YehorFeshchenko/renju-game#5]
https://github.com/YehorFeshchenko/renju-game/blob/8cb6f1dd25e63c998f3f445ef1cc800a3e0572af/main.go#L29C6-L29C21

Is there any way to combine these two methods (checkHorizontal and checkVertical)? they are too similar

**Comment by YehorFeshchenko** [comment]
Good point. To improve readability and maintain clean code practice, two methods were merged into one, `checkLines`.

The issue addressed in changes in this MR: https://github.com/YehorFeshchenko/renju-game/pull/6

**Comment by YehorFeshchenko** [comment]
@IanaKhitsko
